repositories/ActivityRepository.ts

```diff
@@ -7,7 +7,7 @@ import { BaseRepository } from './BaseRepository';
 
 @EntityRepository(ActivityModel)
 export class ActivityRepository extends BaseRepository<ActivityModel> {
-  private static publicActivityTypes = [
+  private static publicActivities = [
     ActivityType.ACCOUNT_CREATE,
     ActivityType.ATTEND_EVENT,
     ActivityType.ATTEND_EVENT_AS_STAFF,
```

Comment:
nit: we can have this be a `Set` so that `isPublicActivityType(type)` is O(1). although the performance boost is most likely miniscule